Documentation/ConsumeNightlyBuild.md

```diff
@@ -16,28 +16,38 @@ To consume nightly builds, create a `NuGet.Config` in your root solution directo
 </configuration>
 ```
 
-### Install packages
+## Install packages
 
-Visual Studio:
+### Visual Studio:
 
 ![File](images/nightly.PNG)
+Example:
```

Comment:
new line needed?

![image](https://user-images.githubusercontent.com/7894084/110208198-f4f03600-7e87-11eb-86cf-7e79eb745c0d.png)

Reply:
I now forced the line break.

![image](https://user-images.githubusercontent.com/19378678/110224146-6b695400-7ed9-11eb-88a9-2503ca9099eb.png)